go/vt/sqlparser/parse_test.go

```diff
@@ -1107,25 +1107,25 @@ var (
 		output: "set /* mixed list */ @@a = 3, names 'utf8', charset 'ascii', @@b = 4",
 	}, {
 		input:  "set session transaction isolation level repeatable read",
-		output: "set @@session.transaction_isolation = 'repeatable-read'",
+		output: "set @@session.transaction_isolation = 'REPEATABLE-READ'",
```

Comment:
I think we want to add more tests so we have both cases, not change the existing ones

Reply:
Here the output changes not the input.

Reply:
This is not longer valid.